framework/encode/vulkan_capture_manager.cpp

```diff
@@ -1066,19 +1068,25 @@ VulkanCaptureManager::OverrideCreateRayTracingPipelinesKHR(VkDevice
                                                            const VkAllocationCallbacks*             pAllocator,
                                                            VkPipeline*                              pPipelines)
 {
+    std::unique_ptr<DeferredOperationCreateRayTracingPipelines>&& deferred_operation_instance =
```

Comment:
I think the use of an rvalue-reference here is confusing.  Is there a reason this can't just be a `shared_ptr`?

Reply:
ok, will change to shared_ptr.